web/src/utils/format.ts

```diff
@@ -0,0 +1,17 @@
+import moment from "moment";
+
+export const dateTimeFormat = (date: Date, format = "YYYY-MM-DD hh:mm") => {
+  return `${moment(date).format(format)}`;
+};
+
+export const bytesFormat = (bytes: number, decimals = 2) => {
```

Comment:
Same here. When/where is this used?

Reply:
I am sorry, bytesFormat is used in organisms not in molecules. I will remove it from this PR and then add it to the "organisms and pages" PR.